### ISSUE TITLE
Welcome Readme + other docs distributed

### ISSUE THREAD

**Issue by jbenet** [jbenet/go-ipfs#580]
We should include several simple docs in a small hash

- [x]  `readme` - the welcome message
- [x] `contact` our contact info
- [x] `help` lists some helpful resources
- [x] `security-notes` see below
- [x] `docs` a directory of ipfs docs themselves!

The `readme` (previously, the welcome message):
- [x] should move to `ipfs cat <simple-docs>/readme` 
- [x] it should include a warning label (see below)
- [x] it should let users know about `-h` and `--help` in all commands
- [x] it should no longer direct people to `ipfs tour`
- [x] remove tour from command registry

```
 ---------------------------------------------------------------
| Warning: this is alpha software - use at your own discretion! |
| To read the security notes by enter:                          |
|   ipfs cat <hash>/security-notes                              |
 ---------------------------------------------------------------
```

the `contact` doc should include:

```
Contact the ipfs dev team:
- Bugs: https://github.com/jbenet/go-ipfs/issues
- Help: irc.freenode.org/#ipfs
- Email: dev@ipfs.io
```

The `security-notes` doc should be:
```
                    IPFS Alpha Security Notes

We try hard to ensure our system is safe and robust, but all software 
has bugs, specially new software. This distribution is meant to be an 
alpha preview, don't use it for anything mission critical.

- this is alpha software and has not been audited. it is our goal
  to conduct a proper sec audit once we close in on a 1.0 release.

- ipfs is a networked program, and may have serious vulnerabilities.
  It is written in Go, and we do not exec.Cmd any user provided data.
  But please point any problems out to us in a github issue, or email
  security@ipfs.io privately.

- ipfs uses encryption for all comm, but it's NOT PROVED SECURE YET! 
  It may be totally broken. For now, the code is included to make sure 
  we benchmark our operations with encryption in mind. In the future, 
  there will be an "unsafe" mode for high performance intranet apps. 
  If this is a blocking feature for you, please contact us.

- We hash and sign all our releases. <include description of how to
  check distribution here>
```

**Comment by whyrusleeping** [comment]
I think that all these files should be in the same directory, and the hash for that directory should be listed in the 'welcome' file that `ipfs init` tells you to cat.

**Comment by jbenet** [comment]
> I think that all these files should be in the same directory, and the hash for that directory should be listed in the 'welcome' file that ipfs init tells you to cat.

Yep! agreed!

**Comment by whyrusleeping** [comment]
is dev@ipfs.io a mailing list? or a single email account?

**Comment by jbenet** [comment]
@whyrusleeping it will be a mailing list and you will be on it.

**Comment by whyrusleeping** [comment]
I can tackle adding all of these files if youre busy with other things.

**Comment by jbenet** [comment]
@whyrusleeping sure go for it. I think we should put all of these under a dir like this in our codebase:

```
assets
└── init-doc
    ├── contact
    ├── docs
    │   └── index
    ├── help
    ├── readme
    └── security-notes

2 directories, 5 files
```

i kicked it off at: https://github.com/jbenet/go-ipfs/compare/master...init-docs

**Comment by whyrusleeping** [comment]
Were going to want to have all the text embedded in the binary. So the working copies can be in assets, and ill work on a makefile that will generate go source files with constants for the text.

**Comment by jbenet** [comment]
Fixed in #639
